lib/rubocop/cop/style/empty_line_after_guard_clause.rb

```diff
@@ -73,6 +74,15 @@ def next_line_rescue_or_ensure?(node)
           parent.nil? || parent.rescue_type? || parent.ensure_type?
         end
 
+        def next_sibling_parent_empty_or_else?(node)
+          next_sibling = node.parent.children[node.sibling_index + 1]
+          return true if next_sibling.nil?
```

Comment:
Why is this cop `Style/EmptyLineAfterGuardClause` and not `Style/EmptyLinesAroundGuardClause`. 😕

Reply:
This cop is responsible for checking that there is a blank line after guard clause (not around), so I think that this name is good 😃 
cc/ @unkmas

Reply:
@koic :+1:

Reply:
I'm not saying the name is wrong, I'm saying that only checking after the guard clause seems like only "half a cop." 🙂

Reply:
@Drenmi I don't think, that before guard clause always should be a blank line.

```ruby
def foo
  result = SomeAction.peform
  return 'WTF' if result.failure?

  result.payload
end
```

This piece of code feels very natural for me.

Reply:
@unkmas: That's fine. It should be configurable to everyone's own preference. But by naming the cop like this, I can't add configuration options to it. 🙂 I would need to add another, `Style/EmpyLineBeforeGuardClause` that duplicates the implementation.

Reply:
I see. I understand about the question of `Style/EmptyLinesAroundGuardClause`.
However, I think that the style of putting a blank line before guard clause feel a bit iffy 💦
In the case of this cop, I think it is important to focus only on after of guard clause.

Reply:
Well. There are as many "feelings" about code as there are developers, which is why the vision of RuboCop has shifted towards being fully configurable. 🙂